Makefile

```diff
@@ -10,15 +10,15 @@ dev-setup:
 
 .PHONY: tests ## Run unit tests
 tests:
-	py.test graphene_django --cov=graphene_django -vv
+	pytest graphene_django --cov=graphene_django -vv
 
 .PHONY: format ## Format code
 format:
```

Comment:
side suggestion here: perhaps we add back `black` in the `dev_requires` dependencies in `setup.py` (so `dev-setup` in this Makefile also installs black, since that will likely be useful for folks' editor configurations just like `ruff`)